src/Core/AdminConsole/Models/Data/EventIntegrations/WebhookIntegrationConfiguration.cs

```diff
@@ -2,4 +2,4 @@
 
 namespace Bit.Core.AdminConsole.Models.Data.EventIntegrations;
 
-public record WebhookIntegrationConfiguration(string url);
+public record WebhookIntegrationConfiguration(string? Scheme, string? Token, string Url);
```

Comment:
🎨 I like to think of optional parameters as followers to required ones, and while it isn't necessarily impactful here the `Url` would be first.